curp/src/client.rs

```diff
@@ -926,6 +1065,11 @@ fn superquorum(nodes: usize) -> usize {
         .wrapping_add(1)
 }
 
+/// Check whether the status is caused by wrong cluster version
+fn is_wrong_cluster_version(status: &tonic::Status) -> bool {
```

Comment:
Do not use the message to tell apart different statues with the same code since the message is unstable. The message may be changed in the future version. If you really need to do so, please add an error label in metadata.